countryCodeToTimeZones.go

```diff
@@ -0,0 +1,2261 @@
+package libphonenumber
+
+import "strconv"
+
+// TODO(ttacon): do we need to worry about load factor?
```

Comment:
I would been somewhat concerned. Could you test some build times with and w/o this file?